backend/icpy/services/filesystem_service.py

```
@@ -1241,6 +1331,102 @@ async def validate_path(self, path: str) -> bool:
+                    merged = {**prev, **payload}
+                else:
+                    merged = payload
+                merged['timestamp'] = time.time()
+                self._event_buffer[composite] = merged
+                # Trim buffer if oversized
+                if len(self._event_buffer) > self._event_batch_max_size:
+                    # Drop oldest arbitrary items
+                    for _ in range(len(self._event_buffer) - self._event_batch_max_size):
+                        try:
+                            self._event_buffer.pop(next(iter(self._event_buffer)))
+                        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)